### PR TITLE
[ZEPPELIN-792]Misspelt words and Wrong method parameters

### PR DESCRIPTION
**Pull request by zuny25**
### What is this PR for?
Fix misspelt words and wrong method parameter on zeppelin-interpreter
https://issues.apache.org/jira/browse/ZEPPELIN-792

### What type of PR is it?
Bug Fix

### Todos
 * [x] - Fix misspelt variable
 * [x] - Fix wrong method parameter

### What is the Jira issue?
* [ZEPPELIN-792](https://issues.apache.org/jira/browse/ZEPPELIN-792)

### How should this be tested?
 1. Edit Interpreter dep package
 2. Run DependencyResolverTest

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/AbstractDependencyResolver.java [hunk 1]
```diff
@@ -52,8 +52,8 @@ public void addRepo(String id, String url, boolean snapshot) {
     synchronized (repos) {
       delRepo(id);
       RemoteRepository rr = new RemoteRepository(id, "default", url);
-      rr.setPolicy(true, new RepositoryPolicy(
-          snapshot,
+      rr.setPolicy(snapshot, new RepositoryPolicy(
```
Comment (minahlee) [inline zeppelin-interpreter/src/main/java/org/apache/zeppelin/dep/AbstractDependencyResolver.java]: I believe just `new RepositoryPolicy()` will do the same thing

**Comment by minahlee** [comment]
LGTM

**Comment by corneadoug** [comment]
@zuny25 Could you rebase this one and re-trigger the CI?

**Comment by bzz** [comment]
@zuny25 thank you for contribution!
CI is green now. 
Looks good to me.

**Comment by corneadoug** [comment]
Merging if there is no more discussions

**Pull request closed without merging**